bulkcopy.go

```diff
@@ -323,6 +324,19 @@ func (b *Bulk) makeParam(val DataValue, col columnStruct) (res param, err error)
 		return
 	}
 
+	switch valuer := val.(type) {
```

Comment:
>switch valuer := [](http://example.com/codeflow?start=1&length=16)

should this switch be invoked after the switch instead of before it? Is there no implementation of `Valuer` that's not already handled directly by the switch?